.github/ISSUE_TEMPLATE/icon_update.yml

```
@@ -0,0 +1,58 @@
+name: Icon update
+description: Help us improve by reporting outdated icons
+title: "Update: "
+labels: [icon outdated]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `icon outdated` label][icon-outdated-issues]
+        (including closed issues) before you continue. If you find one for the brand
+        you're reporting then leave a comment on it or add a reaction.
+
+
+        [icon-outdated-issues]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22icon+outdated%22+is%3Aopen
+
+  - type: input
+    attributes:
+      label: Brand Name
+      placeholder: "Example: Simple Icons"
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Official resources for icon and color
```

Minor, for consistency use title case

```suggestion
      label: Official Resources for Icon and Color
```